apps/amakrushi/src/components/MorePage/FeedbackPage/index.tsx

```diff
@@ -18,11 +20,20 @@ const MorePage: React.FC = () => {
         <div className={styles.rating}>
           <h1>Did you find this useful?</h1>
           <div className={styles.stars}>
-            <FaStar />
-            <FaStar />
-            <FaStar />
-            <FaStar />
-            <FaStar />
+          {Array.from({ length: 5 }, (_, index) => {
+              if (index + 1 <= rating) {
+                return (
+                  <FaStar onClick={() => setRating(index + 1)} key={index} />
+                );
+              } else {
```

Comment:
just a question, are we allowed to re-provide the rating for the app? because you are always allowing us to update the rating